debug:                          false   # bool, specifies whether or not to show the unvalid TLs
# First
first_pseudoPosition_offset:    0.01    # float [m]

# Search
max_num_cones_to_consider:      10      # int
min_angle_loop:                 0.4     # float [rad]

# Heuristic
max_radius_to_next_cone:        5.5     # float [m]
dist_ponderation:               0.6     # float [_], assume angle_ponderation = 1-dist_ponderation

# Closure
min_trace_loop_length:          10      # int [num]

# Short
min_angle_short:                1.74    # float [rad]
max_short_trace_length:         7       # int [num]

# Validation Conditions
min_percentage_of_cones:        90.0    # float [%]
max_distSq_betw_trace_centrd:   25.0    # float [mÂ²]